reo/src/pvwatts.py

```diff
@@ -134,8 +134,8 @@ def __init__(self,
             # if the site is in the southern hemisphere, and no tilt has been specified,
             # then set the tilt to the positive latitude value and change the azimuth to zero
             if self.latitude < 0:
```

Comment:
do we still need this logic in pvwatts.py (the whole if/else block)? It looks like it is taken care of in the PV Tech class

Reply:
No I think this was duplicative. Addressed in https://github.com/NREL/REopt_API/pull/326/commits/c9e51e3646b1365d8301d6e54450ab60e7db7b64